shenyu-client/shenyu-client-dubbo/shenyu-client-alibaba-dubbo/src/main/java/org/apache/shenyu/client/alibaba/dubbo/AlibabaDubboServiceBeanListener.java

```diff
@@ -78,7 +79,8 @@ public AlibabaDubboServiceBeanListener(final ShenyuRegisterCenterConfig config,
         this.appName = appName;
         this.host = props.getProperty("host");
         this.port = props.getProperty("port");
-        executorService = new ThreadPoolExecutor(1, 1, 0L, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<>());
+        executorService = new ThreadPoolExecutor(1, 1, 0L, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<>(), new ThreadFactoryBuilder()
```

Comment:
I'm a little confused. What's the difference between this and `java.util.concurrent.Executors#newSingleThreadExecutor(java.util.concurrent.ThreadFactory)` ?

Reply:
Maybe someone watched alibaba p3c before.

Reply:
finished. please review.

Reply:
Thanks. I'm not quite sure that using only one thread is a reasonable choice. @yu199195 @tydhot

Reply:
.... why not used Executors.newXXXX ?